### ISSUE TITLE
Upload: file selection

### ISSUE THREAD

**Issue by dlowenberg** [CDL-Dryad/dryad-product-roadmap#591]
Screen reader repeats phrase “No files selected” even when files have been selected.

Remove links aren’t explicitly associated with files. | Pending files should be put in a table so that users can tell which files go with which Remove links.
